english/marshmallow.md

```diff
@@ -538,8 +538,7 @@ If you eat a lot, you should also [exercise](exercise/exercise.md)
 
 This [miracle man] (copper-plate-man/copper-plate-man.md) can read your mind without uttering a word.
 
-You decide that you do not want to visit the miracle man. [You check your 
-phone to find something else to do](check-phone/check-phone.md).
+You decide that you do not want to visit the miracle man. [You check your phone to find something else to do](check-phone/check-phone.md).
```

Comment:
This line is longer than 120 characters. Could you break it up into multiple lines? That way it will be easier to read the raw files or view diffs on the command line.  No need to add blank lines in between, just press enter somewhere in the middle of the line.